packages/libs/deer/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+# Please also update the badges in `README.md`, `src/lib.rs`, and `macros/`
```

Comment:
Appreciate this is a mistake in the `error-stack` one but lets do it here anyway

```suggestion
# Please also update the badges in `README.md`, `src/lib.rs`, and `macros/` when changing this
```